README.md

```diff
@@ -1,27 +1,35 @@
-# CISA Vulnrichment 
+# CISA Vulnrichment
 
 The CISA Vulnrichment project is the public repository of CISA's enrichment of public CVE records through CISA's ADP (Authorized Data Publisher) container. In this phase of the project, CISA is assessing new and recent CVEs and adding key [SSVC](https://www.cisa.gov/stakeholder-specific-vulnerability-categorization-ssvc) decision points. Once scored, some higher-risk CVEs will also receive enrichment of [CWE](https://cwe.mitre.org/), [CVSS](https://www.first.org/cvss/), and [CPE](https://csrc.nist.gov/publications/search?keywords-lg=CPE) data points, where possible.
 
-Producers and consumers of this CVE data should already be familiar with the current [JSON format](https://www.cve.org/Media/News/item/blog/2023/03/29/CVE-Downloads-in-JSON-5-Format), and can access this data in the normal ways, including the [GitHub API](https://docs.github.com/en/rest/quickstart).
+Producers and consumers of this CVE data should already be familiar with the current [CVE REcord Format](https://www.cve.org/AllResources/CveServices#CveRecordFormat) and can access this data in the normal ways, including the [GitHub API](https://docs.github.com/en/rest/quickstart).
```

Comment:
The text here has REcord instead of Record (see the capital E)

Reply:
Fixed!